### PR TITLE
improved planet axes

### PR DESCRIPTION
**Pull request by gzotti**
AT LAST!

I want to improve the orientation and rotation of planets, especially the Moon. 
Best data for planet rotation come from the IAU WGCCRE09 report, and later updates of that series. However, these data are not compatible to the way Stellarium has computed rotations so far.

It seems this is now compatible to the rest of the world, at least when we have the appropriate data.

The old solution is still used if such data are not present.

There is still some need to cleanup, but functionally it seems good now!

Q: I had implemented Lunar libration from Astronomical Algorithms. This is now unused. Keep as reference or delete?

Related: #347, #357, #358 #877

### CODE REVIEW

**Comment by alex-w** [comment]
@gzotti this is may suggest the solution for Moon’s orientation problem: https://github.com/pchev/skychart/blob/98b7ac40b660beb6acd33d3d183e401e9fe76388/skychart/cu_planet.pas#L1226

File: src/core/modules/NomenclatureItem.cpp [hunk 1]
```diff
@@ -227,6 +227,7 @@ QString NomenclatureItem::getInfoString(const StelCore* core, const InfoStringGr
 		QString description = getNomenclatureTypeDescription(nType, planet->getEnglishName());
 		if (getNomenclatureType()!=NomenclatureItem::niUNDEFINED && !description.isEmpty())
 			oss << QString("%1: %2").arg(q_("Landform description"), description) << "<br />";
+		oss << QString("%1: %2").arg(q_("Solar altitude")).arg(QString::number(getSolarAltitude(core), 'f', 1)) << "<br />";
```
Comment (alex-w) [inline src/core/modules/NomenclatureItem.cpp:230]: what about units?

Comment (gzotti) [reply src/core/modules/NomenclatureItem.cpp]: Ah sure. Added.

**Review by alex-w — APPROVED** [review]

**Comment by gzotti** [comment]
I had implemented Lunar libration from Astronomical Algorithms before I did the internal (better) solution. This is now unused. Keep as reference or delete these files and the commented-away use in Planet.cpp? I now tend to remove them.

**Review by alex-w — APPROVED** [review]
Deleting "old" libration is OK for me

**Pull request merged**